.gitignore

```diff
@@ -10,6 +10,9 @@
 # production
 /build
 
+# Submodules
+angular/angular-showcase-demo
```

Comment:
You shouldn't need to put a submodule in the gitignore...git ignores changes made in the submodule by default. I'm not sure if this will have unintended results.

Reply:
Ah okay, I've removed it from the gitsubmodule